Add verysmart plugin and data

mdawsondev opened the pull request:
Added verysmart.py to plugins and verysmart.txt to data. Quotes are picked at random.

Quotes pulled from [r/iamverysmart](https://reddit.com/r/iamverysmart) and peppered with a bit of Jaden Smith and Trump Twitter quotes.  Didn't include any vulgarity so it's SFW. Quotes are lexicographically sorted and written verbatim as they were posted (grammatical errors and punctuation included).

I threw everything into the .txt so it should be easy to add or remove content as desired. This adds the commands `.vs` `.verysmart` `.smart` `.iamverysmart` and `.iavs`. Tested on snoonet.

Commit:
Add verysmart plugin and data

Commit:
Add final new line to pass CodeFactor test

linuxdaemon commented on plugins/verysmart.py:
```diff
@@ -0,0 +1,12 @@
+
+@hook.command('smart', 'verysmart', 'vs', 'iamverysmart', 'iavs', autohelp=False)
+def smart(bot):
+    with codecs.open(os.path.join(bot.data_dir, "verysmart.txt"), encoding="utf-8") as fp:  
```
Load the data once in an `on_load` hook and just access that from the command., Opening the file every time someone runs a command is not a good idea.

Commit:
Reformat to preload quote data via on_start

Commit:
Move vsquotes and refactor to extend

linuxdaemon commented on plugins/verysmart.py:
```diff
@@ -0,0 +1,17 @@
+        vsquotes.extend(quote.strip() for quote in fp.readlines())
+
+@hook.command('smart', 'verysmart', 'vs', 'iamverysmart', 'iavs', autohelp=False)
+def verysmart():
```
Please add a docstring in the format 
```python
"""<required arg> [optional arg] - command description"""
```

linuxdaemon committed:
Merge branch 'gonzobot' into gonzobot

Commit:
Add docstrings to functions

linuxdaemon commented on plugins/verysmart.py:
```diff
@@ -0,0 +1,19 @@
+
+@hook.command('smart', 'verysmart', 'vs', 'iamverysmart', 'iavs', autohelp=False)
+def verysmart():
+    """Return a random choice from the quote list.""" 
```
Docstrings without any arguments should start with a hyphen.

Commit:
Add hyphen to docstrings

`bot` argument is injected, no arguments accepted by users. Added hyphen to both docstrings.

linuxdaemon committed:
Merge branch 'gonzobot' into gonzobot
